docs/plugins/data.malware-scan.md

```diff
@@ -0,0 +1,26 @@
+data.malware-scan
+==========
+
+This plugin extracts hashs(md5,sha1,sha256) of each attachment and feeds them to MHR based blacklists such as 
+[MALWARE HASH REGISTRY][1].
```

Comment:
The DNS list name is fixed, so unless you offer this as a configurable list, it will only work with the MHR.